src/admin/reducers/search.js

```diff
@@ -1,4 +1,5 @@
 const initialState = {
+  auth: true,
```

Comment:
I bet there could, as the `auth` info should be in the state. I'll give that a go.